src/server/biosecurity-map/upload-progress/index.js

```diff
@@ -1,45 +1,63 @@
 import Wreck from '@hapi/wreck'
-import { Page } from '../../common/model/page/page-model.js'
-import { PageController } from '../../common/controller/page-controller/page-controller.js'
-import { QuestionPage } from '../../common/model/page/question-page-model.js'
+import { BiosecurityAnswer } from '../../common/model/answer/biosecurity-map/biosecurity-map.js'
 import { config } from '~/src/config/config.js'
+import { QuestionPage } from '../../common/model/page/question-page-model.js'
+import { QuestionPageController } from '../../common/controller/question-page-controller/question-page-controller.js'
+import { Page } from '../../common/model/page/page-model.js'
+import { uploadConfig } from '../upload-config.js'
+import { UploadPlanPage } from '../upload-plan/index.js'
 
-/**
- * @import {NextPage} from '../../common/helpers/next-page.js'
- * @import {ConfirmationPayload} from '../../common/model/answer/confirmation/confirmation.js'
- */
-
-class BioSecuritySummary extends QuestionPage {
-  urlPath = '/biosecurity/check-answers'
+class BioSecuritySummary extends Page {
```

Comment:
could move this bit to the actual file that we will use for the summary page?

Reply:
will do in turn